.github/workflows/ci.yml

```
@@ -0,0 +1,98 @@
+name: Django CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

Старайся по возможность использовать самые свежие версии библиотек
https://github.com/actions/checkout
```suggestion
        uses: actions/checkout@v4
```